frontend/src/test/scala/bloop/tasks/IntegrationTestSuite.scala

```diff
@@ -70,7 +73,11 @@ class IntegrationTestSuite(testDirectory: Path) {
         (state, rootProject)
     }
 
-    val reachable = Dag.dfs(state.build.getDagFor(projectToCompile)).filter(_ != projectToCompile)
+    val reachable =
```

Comment:
Why is this necessary?

Reply:
I renamed `state` to `initialState`, and `state` is now the state after cleaning. We need to clean because the project may have already been compiled by a previous test.